workshops/cringe_101/README.md

```diff
@@ -237,22 +241,18 @@ on your site.
 **Steps to add and change fonts:**
 
 1. Navigate to [google.com/fonts](https://www.google.com/fonts).
-2. Choose a font by clicking the **middle** icon (you can see more of the font
-   details by clicking the 1st button → Scroll to go click the middle icon).
-3. Fonts have different weights → Choose a few.
-4. This is just a fancy term for the thickness.
-5. The numbers range from 100 to 700 in hundreds for a total of 7 a. Some fonts
-   only have a few weights, others have all 7.
-6. Go to the number 3 area where it says "Add this code to your website".
-7. You'll notice it looks exactly like a CSS file -- that's because it is!
-8. Since it's a CSS file, you know what that means! Copy the code & paste in
-   your head.
+2. To select a font, click the + on any of the ones on the front page or by searching for one on the top right corner.
+3. Fonts have different styles, scroll down from the font you’ve selected and click the + button.
```

Comment:
I believe you're using the wrong apostrophe in "you've" and that's triggering a spellcheck error.